### DISCUSSION TITLE
Wire:naviate is not working when using laravel debugbar with livewire.esm.js

### DISCUSSION THREAD

**Discussion by sohailahmad07** [livewire/livewire#6097] [7. Bugs (archived)]
### Livewire version

3.0.0-beta.5

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

First, i don't know if the bug is due to debugBar or livewire but here is how you can reproduce it 

1. install livewire v3 & laravel debug bar
2. import the alpine in app.js to use livewire.esm.js **IMPORTANT** 
3. wire navigate throw some kind of attribute error 

### Additional information

code in app.js 
```
import {Alpine, Livewire} from '../../vendor/livewire/livewire/dist/livewire.esm.js'
// here i am using  `Alpine.store` for switching between light and dark mode
Livewire.start()
```

app.js
```
@livewireScriptConfig
```

if i uninstall debugbar Or Remove the above code wire navigate start working again

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PhiloNL** [comment]
What is the error you get?

**Reply by sohailahmad07** [reply]
hi below is the error i get in console log 

```
Uncaught ReferenceError: assignment to undeclared variable attr
    cloneScriptTag http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:6814
    prepNewBodyScriptTagsToRun http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:6779
    prepNewBodyScriptTagsToRun http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:6773
    swapCurrentPageWithNewHtml http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:6751
    navigateTo http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:6878
    preventAlpineFromPickingUpDomChanges http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:6917
    navigateTo http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:6876
    getPretchedHtmlOr http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:5159
    fetchHtmlOrUsePrefetchedHtml http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:6911
    navigateTo http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:6872
    src_default http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:6866
    handler http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:5175
    whenThisLinkIsPressed http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:5178
    src_default http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:6865
    whenThisLinkIsPressed http://127.0.0.1:5173/vendor/livewire/livewire/dist/livewire.esm.js:5172

```

**Reply by rahulhaque** [reply]
Hi @sohailahmad07, can you try after adding `/livewire/update` to the `exception` array of laravel debugbar config and see if it helps.

**Reply by LukasJK** [reply]
Same error occurs on my end.

debugbar.php
`'except' => [
        'telescope*',
        'horizon*',
        'livewire/update',
    ],`

I guess it's because debugbar has script entries it injects into the page.

**Reply by sohailahmad07** [reply]
i think so too cause i install clockwork and every is fine

**Comment by LukasJK** [comment]
Anyone had any luck in figuring it out? Still have this with 3.0.2
